### PR TITLE
Implement manual-change subcommand

### PR DESCRIPTION
**Pull request by DavidS-om**

### CODE REVIEW

**Comment by DavidS-om** [comment]
All those alerts would be fixed by https://github.com/github/codeql/pull/13835

File: cmd/manualchange.go [hunk 1]
```diff
@@ -0,0 +1,388 @@
...
+					"responders":               status.Summary.Responders,
+					"queriesSent":              queriesSent,
+					"post_processing_complete": status.PostProcessingComplete,
+				}
+
+				if status.Done() {
+					// fall through from all "final" query states, check if there's still queries in progress;
+					// only break from the loop if all queries have already been sent
+					// TODO: see above, still needs DefaultStartTimeout implemented to account for slow sources
+					allDone := allDone(ctx, activeQueries, lf)
```
Comment (dylanratcliffe) [inline cmd/manualchange.go:240]: Realistically this check shouldn't be required as the gateway shouldn't tell us we're done unless *everything* is done right? Can we change maybe the "all responders done, with unfinished queries" log to an error so it's easier to spot? Ideally if this every happens we'd know about it

**Review by dylanratcliffe — APPROVED** [review]
Approved but with one comment

File: cmd/manualchange.go (hunk 1 above)
Comment (DavidS-om) [reply cmd/manualchange.go:240]: `allDone` has a `Debugf("query still active")` for exactly that.

**Pull request merged**